src/App.tsx

```diff
@@ -15,71 +16,85 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
```

Comment:
```suggestion
interface State {
```
Not critical, but it is better to use interfaces for object typing

Reply:
agree